targets/PandABox/target-top.dts

```diff
@@ -80,3 +80,9 @@
 		};
 	};
 };
+
+/ {
+	pandainfo {
+		version = "GIT_VERSION";
+	};
+};
```

Comment:
@EmilioPeJu where would you put this in the device tree?

Reply:
An alternative approach would be hanging version from the root node, for example:
```
/ {
	model = "PandABox";
	version = "GIT_VERSION";
	....
}```
The approach of adding a pandainfo node is also acceptable, specially if we plan to add more information properties in future.
In practices, the only thing that changes is if you read `/proc/device-tree/version` or `/proc/device-tree/pandainfo/version` to get the value.